light-zk.js/src/idls/verifier_program_one.ts

```diff
@@ -59,9 +59,14 @@ export type VerifierProgramOne = {
         "At successful verification protocol logic is executed."
       ],
       "accounts": [
+        {
+          "name": "verifierState",
+          "isMut": false,
+          "isSigner": false
+        },
```

Comment:
the change in the order will mess up the indexer, best put verifierState at the bottom of the account struct

Reply:
I tried my best to minimize the diff in account orders in the latest push. And the approach is to apply the fields from macro first and then fields from the original struct.

But we will have to change the order in indexer anyway. The reasons being:

* I had to group all SOL- and SPL-related fields together, otherwise the macro code would become quite ugly (repeating the same if-s just to paste those fields in multiple places).
* Currently (in the main branch) we put event_merkle_tree at the end aftter verifier_state. We cannot do it here - event_merkle_tree is part of the macro, verifier_state is a field from the original struct.